CAP/gap/WrapperCategory.gd

```diff
@@ -6,6 +6,48 @@
 
 #! @Chapter Create wrapper hulls of a category
 
+#! The support for towers of category constructors is one
+#! of the main design features of CAP. Many categories that
+#! appear in the various applications can be modeled by such towers.
+#! The wrapper category constructor allows adding
+#! one last layer on top which allows specifing
+#! the desired (re)interprateation of such a modeling tower.
+#! In particular, the wrapper category constructor allows specifying
+#! customized methods for the operations
+#! * ObjectConstructor
+#! * MorphismConstructor
+#! * ObjectDatum
+#! * MorphismDatum
+#! * View/Display/Print for the category/objects/morphisms
+#! in order to reflect the desired interpretation and with
+#! a user-interface that is independent of the modeling tower.
+#! Note that the same tower might have multiple interpretations.
+#! 
+#! <Table Align="|c|">
+#! <Caption>A tower of categories modeling the category <C>W</C></Caption>
+#! <HorLine/>
+#! <Row>
+#!   <Item><C>W := WrapperCategory( cat_n )</C></Item>
+#! </Row>
+#! <HorLine/>
+#! <Row>
+#!   <Item><C>cat_n := CategoryConstructor_n( cat_{n-1} )</C></Item>
+#! </Row>
+#! <HorLine/>
+#! <Row>
+#!   <Item>...</Item>
+#! </Row>
+#! <HorLine/>
+#! <Row>
+#!   <Item><C>cat_1 := CategoryConstructor_1( non_categorical_input )</C></Item>
+#! </Row>
+#! <HorLine/>
+#! </Table>
```

Comment:
There might be towers with multiple roots, e.g. in the context of `FunctorCategories`. Of course that is harder to visualize, but I think it should be at least mentioned in the text.